lib/deliver/commands_generator.rb

```diff
@@ -49,7 +49,22 @@ def run
           Deliver::Runner.new(options).run
         end
       end
+      command :submit_build do |c|
+        c.syntax = 'deliver submit_build'
+        c.description = 'Submit a specific build-nr for review, use latest for the latest build'
+        c.action do |args, options|
+          options = FastlaneCore::Configuration.create(Deliver::Options.available_options, options.__hash__)
+          options.load_configuration_file("Deliverfile")
+
+          options[:skip_metadata] = true
+          options[:skip_screenshots] = true
+          options[:submit_for_review] = true
+          options[:skip_upload] = true
```

Comment:
Sorry, but I had to rename this option https://github.com/fastlane/deliver/commit/8097b7ab3a87f2e6695d98faeed32b28a1a176cb